src/System.Linq/tests/ConsistencyTests.cs

```diff
@@ -22,16 +22,18 @@ public static void MatchSequencePattern()
                 typeof(Enumerable),
                 typeof(Queryable),
                  new[] {
-                     "ToLookup",
-                     "ToDictionary",
-                     "ToArray",
-                     "AsEnumerable",
-                     "ToList",
+                     nameof(Enumerable.ToLookup),
+                     nameof(Enumerable.ToDictionary),
+                     nameof(Enumerable.ToArray),
+                     nameof(Enumerable.AsEnumerable),
+                     nameof(Enumerable.ToList),
                      "Fold",
                      "LeftJoin",
-                     "Append",
-                     "Prepend",
-                     "ToHashSet"
+                     nameof(Enumerable.Append),
+                     nameof(Enumerable.Prepend),
```

Comment:
Nobody had the idea of doing so. At the time I was thinking they wouldn't be a good idea to add, but I think now I was wrong on that.